docs/troubleshoot/cli/cli-use-curl-to-troubleshoot.md

```diff
@@ -0,0 +1,101 @@
+# Using cURL to troubleshoot Zowe CLI
+
+When a REST API call fails on Zowe CLI, you can use cURL to troubleshoot.
+
+Run a command with cURL and compare its output with what is returned using Zowe CLI. This can help pinpoint whether the problem lies with z/OSMF or Zowe CLI, depending on which command returns an API error.
+
+## Understanding cURL commands
+
+A cURL command using the `https` protocol follows the basic syntax `curl <URL>`.
+
+Add cURL options to establish communication between Zowe CLI and z/OSMF, as in the following example command:
+
+```
+curl --location --request <API method> "https://<host>:<port>/<API>" --header "X-CSRF-ZOSMF-HEADER;" --insecure --user "<ID>:<PASSWORD>"
+```
+**NOTE:** Some terminals may require single quotes rather than double quotes.
+
+
+### `--location`
+
+Use `--location` to allow the server to redirect to a different URL, if needed. 
+
+When a different location exists and `--location` is not included in the command, the server responds with a 3XX status code.
+
+### `--request <API method>`
+
+Use `--request` to identify the API method and name. Not necessary when the API method is `GET`.
+
+- `<API method>`: Specifies the API method used in the request.
+
+### `https://<host>:<port>/<API>"`
+
+Indicates the protocol and URL.
+
+- `<host>`: Specifies the host name where the z/OSMF services are running.
+- `<port>`: Specifies the REST port number.
+- `<API>`: Specifies the name of the API used in the request.
+
+### `--header "X-CSRF-ZOSMF-HEADER;"`
+
+Required to establish communication with z/OSMF. Specifies that the client is sending a cross-site request to the REST interface.
+
+- `;`: Indicates that the header has no value. (A value is not required.)
```

Comment:
We could add a note about how to pass headers with values like this:
> If you need to pass additional headers with values, use a colon to separate the key and value. For example: `--header "X-IBM-Data-Type: binary"`

Reply:
Are values actually needed for these cURL commands? If not, then I don't think we need to explain how to include values.

Reply:
No values are needed for the cURL command to check z/OSMF status in the example below. But header values may be required for other cURL commands. For example to upload a binary file using the z/OSMF API you would need to provide the `"X-IBM-Data-Type: binary"` header.